src/components/ui/Link/index.js

```diff
@@ -44,13 +46,14 @@ const Link = (props) => {
       {children}
       {external && (
         <React.Fragment>
-          {' '}
           <Icon
             name="external-link"
-            title="External link"
+            noMargin
             small
+            strokeColor={theme.colors.neutral.grey600}
+            title="External link"
             css={css`
-              margin: 0;
+              margin-left: 0.4rem;
```

Comment:
We'll want to investigate our RTL-habits eventually, but this is fine for now.

(Should this be a theme value?)

Reply:
GODDAMN IT YOU ARE PICKY. 

But yes, it should, and now it is. 

+1 on the RTL, although here it'll obviously be a bit weirder since the icon is just stuck at the end of the link, rather than being CSSed there. RTL might need to be a whole sweep we do at some point to figure out how to best accomodate it.